core/spec/models/mno_enterprise/user_spec.rb

```diff
@@ -373,6 +372,35 @@ def reload_user
       end
     end
 
+    describe '.activate_otp' do
+      let(:user) { build(:user) }
+      before(:each) do
+        user.stub(:save!)
+        MnoEnterprise::User.stub(:find) { [user] }
+      end
+
+      it "sets self.attributes['otp_required_for_login'] to true" do
+        user.attributes = {}
+        user.activate_otp
+        expect(user.attributes['otp_required_for_login']).to be true
```

Comment:
Doesn't `user.otp_required_for_login` work? 
Also the description should be more user friendly:
```ruby
it "sets #otp_required_for_login to true" do
  user.attributes = {} # Not sure about this though...
  user.activate_otp
  expect(user.otp_required_for_login).to be true
end
```